objects/1b39da/script.lua

```diff
@@ -8,6 +8,7 @@ function Broadcast(params)
     return "Blitz - Remember, Invaders get an additional set of Actions at the end of Setup"
 end
 
+modifyCostPriority = 10
```

Comment:
we may want to add an extra 0 to these just to allow for more room in between things

Reply:
i dont envision us needing that many, but kind of hard to go back later and change things here since it would break custom content

Reply:
Priorities don't have to be integers, or even positive, so until we hit floating point precision limits, there'll always be more room to slip things in. If it weren't for that, I'd be slightly uncomfortable relying on this system at all. I can still make them bigger if you want. I also had half a mind to mention that negatives/fractions are permitted in the documentation; I can toss that in as well if you'd like.